spec/controllers/locations/floors_controller_spec.rb

```diff
@@ -93,29 +92,34 @@ module Locations
           expect {
             post :create, {:floor => valid_attributes}, valid_session
```

Comment:
The create and update routes all require a ```:library_id``` param. 
```post :create, {:library_id => valid_attributes[:locations_library_id], :floor => valid_attributes}, valid_session``` allows the application accept the route. After that another error comes up about a ```floor_url``` method for the floor model.